docs/migration.md

```diff
@@ -281,3 +281,95 @@ metadata:
     name: fio-cstor
     openebs.io/target-affinity: fio-cstor
 ```
+
+# Migrating jiva External Provisioned volumes to jiva CSI volumes (Experimental)
+
+These instructions will guide you through the process of migrating jiva volumes from the old v1apha1 external provisioned spec to v1 CSI spec. 
+
+### Prerequisites
+
+Before migrating the volumes make sure the following prerequisites are taken care of:
+
+ - The jiva-operator should be installed with version 2.7.0 or above. You can install the correct version of jiva-operator from [charts](https://github.com/openebs/charts/tree/gh-pages). Get the jiva-operator yaml within the correct versioned folder and install. 
+ - **The application needs to be scaled down before migrating.** This is required as the old PVC and PV spec needs to be deleted at the end of migration.
+ 
+### Migration steps
+
+1. Scale down volume controller and all replica deployments. You can find these deployments using the command:
+   ```
+    $ kubectl -n openebs get deploy
+    NAME                                             READY   UP-TO-DATE   AVAILABLE   AGE
+    pvc-898d90cc-ec73-4868-9205-1a8ba141a5bb-ctrl    1/1     1            1           50s
+    pvc-898d90cc-ec73-4868-9205-1a8ba141a5bb-rep-1   1/1     1            1           50s
+   ```
+
+2. Create a jivaVolumePolicy equivalent to old storageclass annotations. For example if the original storageclass was:
+   ```yaml
+    apiVersion: storage.k8s.io/v1
+    kind: StorageClass
+    metadata:
+      name: jiva-1r
+      annotations:
+        openebs.io/cas-type: jiva
+        cas.openebs.io/config: |
+          - name: ReplicaCount
+            value: "1"
+    provisioner: openebs.io/provisioner-iscsi
+   ```
+    The equivalent jivaVolumepolicy will look like:
+
+    ```yaml
+    apiVersion: openebs.io/v1alpha1
+    kind: JivaVolumePolicy
+    metadata:
+      name: example-jivavolumepolicy
+      namespace: openebs
+    spec:
+      replicaSC: openebs-hostpath
+      target:
+        replicationFactor: 1
```

Comment:
Could you add a comment here so that it is clear that `target.replicationFactor` value should be the same as the value to `ReplicaCount` from the storageClass?